src/Dhall/Import.hs

```diff
@@ -103,6 +104,8 @@ module Dhall.Import (
       exprFromPath
     , load
     , loadWith
+    , load'
+    , loadWith'
```

Comment:
What do you think about just exporting two load functions:

* `load` (with no custom context and runs in `IO`)
* `loadWith` (with custom context and runs in `MonadCatch`, which you currently call `loadWith'`)

My reasoning here is to reduce API proliferation.  Also, people who want to customize the context will probably also want to customize the path import logic, too

This would be a breaking change (since it changes the type of `loadWith` which is already part of the public API), but I'd be fine with that since the next release has a minor breaking change anyway